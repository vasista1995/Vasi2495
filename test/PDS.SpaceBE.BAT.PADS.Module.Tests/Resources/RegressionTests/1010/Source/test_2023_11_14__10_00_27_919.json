{
  "IdSource": "SPACEACT2:BAT:BE:RU243389B26:BE1I:Wire_Bond1:GoldBallHeightBump:68003:240184639:519",
  "Data_flat_Metadata": {
    "Device": "U5435A",
    "ParameterClass": "NON_SPC",
    "CFComment": "FOL",
    "Segment": "WB",
    "FourDReport": "*",
    "ParameterName": "GoldBallHeightBump",
    "ParameterUnit": "um",
    "ParameterOper": "Wire_Bond1",
    "ParameterFacility": "BE1I",
    "SiteKey": "BAT",
    "ChannelId": "68003",
    "SpaceInstanceName": "BE",
    "CreatedTimestampUtc": "2023-11-14T06:03:14Z",
    "CreatedTimestamp": "2023-11-14T13:03:14Z",
    "UpdatedTimestamp": "2023-11-14T13:44:17Z",
    "UpdatedTimestampUtc": "2023-11-14T06:44:17Z",
    "ExportedTimestampUtc": "2023-11-14T13:03:13Z",
    "ExportedTimestamp": "2023-11-14T13:03:13Z",
    "SampleTimestampUtc": "2023-11-14T06:03:13Z",
    "SampleTimestamp": "2023-11-14T13:03:13Z",
    "SourceDataLevel": "C",
    "ChannelName": "Ball_Height/AU/30um/Bump",
    "RvStoreFlag": "Y",
    "CalcCreated": "11/14/2023 13:03:13",
    "ILdsID": 0,
    "SpcComExtern": "CA-Attributes started. Ticket id: 60f88852-d256-4977-b972-bb440088db09\nCA triggered by violation 1 (Raw value above specification) in channel (68003/519)\n====\nTicketId=60f88852-d256-4977-b972-bb440088db09 state=PASS executed OCAP=General, HOLDLOT, STOPEQUIPMENTPROCESSING, QUALITYALARMTRIGGER\nCA triggered by violation -1 (Unknown violation description) in channel (-1/-1)",
    "AcceptFlag": "Y",
    "UpdatedBy": "pobtech",
    "ClOrigin": 0,
    "OutOfOrder": "N",
    "Calc2ID": 2526354,
    "ViolCount": 2,
    "CalculationStrategy": "ND",
    "LdsID": "1010",
    "IsCalcParam": 0,
    "ChannelType": "N",
    "ChState": "Online"
  },
  "Data_1list_RawValues": {
    "Data_flat_Limits": {
      "CkcId": "519",
      "MeasurementSpecLimits": {
        "SpecHighEnabled": "Y",
        "SpecLowEnabled": "Y",
        "SpecHigh": 24.0,
        "SpecLow": 9.0,
        "SpecTarget": 16.5,
        "SpecTargetOrigin": "V",
        "ExtSpecLimEnable": "NN",
        "ExtSpecUpper": 0.0,
        "ExtSpecLower": 0.0,
        "LimitEnable": "YYNYNNYNYYNNNNNNNNNNNNNNNNNNNNNNNNNNN"
      },
      "SpaceAggregates": {
        "Mean": "23,7",
        "Median": "24,1",
        "Sigma": "1,62326830807479",
        "Min": "21",
        "Max": "25,4",
        "Q1": "23,9",
        "Q3": "24,1"
      },
      "ControlLimits": {
        "ctrl_mean_low": 11.25,
        "CtrlMeanLowEnabled": "Y",
        "ctrl_mean_target": 16.5,
        "CtrlMeanTargetEnabled": "Y",
        "ctrl_mean_high": 21.75,
        "CtrlMeanHighEnabled": "Y",
        "CtrlLowEnabled": "N",
        "CtrlSigmaCenterEnabled": "N",
        "CtrlRangeCenterEnabled": "Y",
        "CtrlCenterEnabled": "N",
        "CtrlHighEnabled": "N",
        "CtrlSigmaLowEnabled": "N",
        "CtrlSigmaHighEnabled": "N",
        "CtrlRangeLowEnabled": "N",
        "ctrl_range_target": 0.0,
        "ctrl_range_high": 15.0,
        "CtrlRangeHighEnabled": "Y",
        "ExtMaMV": 20.0675,
        "ExtEwmaMV": 21.6268921073237,
        "ExtMsMV": 3.94565583902093,
        "EwmaS": 1.48350696930272,
        "EwmaR": 3.88954995265765,
        "ExtEwmaMVLCL": 12.8656531284057,
        "ExtEwmaMVCenter": 15.268504,
        "ExtEwmaMVUCL": 17.6713548715943,
        "ExtMaMVLCL": 13.5694318513612,
        "ExtMaMVCenter": 15.268504,
        "ExtMaMVUCL": 16.9675761486388,
        "EwmaSLCL": 0.572474203967793,
        "EwmaSCenter": 1.51113357572865,
        "EwmaSUCL": 2.44979294748951,
        "EwmaRLCL": 1.38119890712411,
        "EwmaRCenter": 3.62764,
        "EwmaRUCL": 5.87408109287589,
        "ExtMSMVLCL": 0.00259778821504498,
        "ExtMSMVCenter": 1.2250389281796,
        "ExtMSMVUCL": 4.92102881621818,
        "MVLal": 0.0,
        "MVUal": 0.0,
        "RawLal": 5.64136422726342,
        "RawUal": 24.8956437727366,
        "SigmaLal": 0.0,
        "SigmaUal": 0.0,
        "RangeLal": 0.0,
        "RangeUal": 0.0
      }
    },
    "Data_1list_RawValues": [
      {
        "SampleId": 240184639,
        "Seqnr": 2,
        "Value": 24.1,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 2,
        "ViolationComments": "RC : EFO PARAMETER TOO HIGH\nCA : ADJUST CURRENT PARAMETER FROM 60 TO 59\nTECH : DITA",
        "ViolationList": "1,3",
        "PrimaryViolation": "1",
        "PrimaryViolationComments": "RC : EFO PARAMETER TOO HIGH\nCA : ADJUST CURRENT PARAMETER FROM 60 TO 59\nTECH : DITA"
      },
      {
        "SampleId": 240184639,
        "Seqnr": 1,
        "Value": 24.1,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 2,
        "ViolationComments": "RC : EFO PARAMETER TOO HIGH\nCA : ADJUST CURRENT PARAMETER FROM 60 TO 59\nTECH : DITA",
        "ViolationList": "1,3",
        "PrimaryViolation": "1",
        "PrimaryViolationComments": "RC : EFO PARAMETER TOO HIGH\nCA : ADJUST CURRENT PARAMETER FROM 60 TO 59\nTECH : DITA"
      },
      {
        "SampleId": 240184639,
        "Seqnr": 3,
        "Value": 21.0,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 2,
        "ViolationComments": "RC : EFO PARAMETER TOO HIGH\nCA : ADJUST CURRENT PARAMETER FROM 60 TO 59\nTECH : DITA",
        "ViolationList": "1,3",
        "PrimaryViolation": "1",
        "PrimaryViolationComments": "RC : EFO PARAMETER TOO HIGH\nCA : ADJUST CURRENT PARAMETER FROM 60 TO 59\nTECH : DITA"
      },
      {
        "SampleId": 240184639,
        "Seqnr": 4,
        "Value": 25.4,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 2,
        "ViolationComments": "RC : EFO PARAMETER TOO HIGH\nCA : ADJUST CURRENT PARAMETER FROM 60 TO 59\nTECH : DITA",
        "ViolationList": "1,3",
        "PrimaryViolation": "1",
        "PrimaryViolationComments": "RC : EFO PARAMETER TOO HIGH\nCA : ADJUST CURRENT PARAMETER FROM 60 TO 59\nTECH : DITA"
      },
      {
        "SampleId": 240184639,
        "Seqnr": 5,
        "Value": 23.9,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 2,
        "ViolationComments": "RC : EFO PARAMETER TOO HIGH\nCA : ADJUST CURRENT PARAMETER FROM 60 TO 59\nTECH : DITA",
        "ViolationList": "1,3",
        "PrimaryViolation": "1",
        "PrimaryViolationComments": "RC : EFO PARAMETER TOO HIGH\nCA : ADJUST CURRENT PARAMETER FROM 60 TO 59\nTECH : DITA"
      }
    ],
    "SampleSize": 5
  },
  "ProductionAction": {
    "Id": "SPACEACT1:BAT:BE:BE1I:Wire_Bond1:GoldBallHeightBump:68003",
    "Type": "SpaceAction"
  },
  "Item": {
    "Id": "Measlot:",
    "IdSystemName": "Measlot",
    "Type": "Lot"
  },
  "SystemLog": {
    "ExtractedTimestampUtc": "0001-01-01T00:00:00",
    "EnqueuedTimestampUtc": "0001-01-01T00:00:00",
    "UniqueSourceKey": "BAT:BE:RU243389B26:BE1I:Wire_Bond1:GoldBallHeightBump:68003:240184639:519"
  }
}