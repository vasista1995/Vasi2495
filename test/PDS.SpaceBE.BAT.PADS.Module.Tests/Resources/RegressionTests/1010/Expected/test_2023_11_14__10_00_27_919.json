{
  "Data_flat_Metadata": {
    "ParameterOper": "Wire_Bond1",
    "ParameterFacility": "BE1I",
    "SiteKey": "BAT",
    "SpaceInstanceName": "BE",
    "BeginTimestampUtc": "2023-11-14T06:03:13Z",
    "BeginTimestamp": "2023-11-14T13:03:13Z",
    "EndTimestampUtc": "2023-11-14T06:03:13Z",
    "EndTimestamp": "2023-11-14T13:03:13Z"
  },
  "Data_flat_StrangeMetadata": {
    "Device": "U5435A",
    "ParameterClass": "NON_SPC",
    "Segment": "WB"
  },
  "Data_1list_Parameters": [
    {
      "Data_1list_RawValues": [
        {
          "Seqnr": 2,
          "SampleId": 240184639,
          "Value": 24.1,
          "is_flagged": "N",
          "SampleTimestamp": "2023-11-14T13:03:13Z",
          "SampleTimestampUtc": "2023-11-14T06:03:13Z",
          "CreatedTimestamp": "2023-11-14T13:03:14Z",
          "CreatedTimestampUtc": "2023-11-14T06:03:14Z",
          "UpdatedTimestamp": "2023-11-14T13:44:17Z",
          "UpdatedTimestampUtc": "2023-11-14T06:44:17Z",
          "NumViolations": 2,
          "ViolationComments": "RC : EFO PARAMETER TOO HIGH CA : ADJUST CURRENT PARAMETER FROM 60 TO 59 TECH : DITA",
          "ViolationList": "1,3",
          "PrimaryViolation": "1",
          "PrimaryViolationComments": "RC : EFO PARAMETER TOO HIGH CA : ADJUST CURRENT PARAMETER FROM 60 TO 59 TECH : DITA",
          "SampleMean": 23.7,
          "SampleMin": 21.0,
          "SampleMax": 25.4,
          "SampleMedian": 24.1,
          "SampleStdev": 1.62326830807479,
          "SampleQ1": 23.9,
          "SampleQ3": 24.1,
          "SampleSize": 5
        },
        {
          "Seqnr": 1,
          "SampleId": 240184639,
          "Value": 24.1,
          "is_flagged": "N",
          "SampleTimestamp": "2023-11-14T13:03:13Z",
          "SampleTimestampUtc": "2023-11-14T06:03:13Z",
          "CreatedTimestamp": "2023-11-14T13:03:14Z",
          "CreatedTimestampUtc": "2023-11-14T06:03:14Z",
          "UpdatedTimestamp": "2023-11-14T13:44:17Z",
          "UpdatedTimestampUtc": "2023-11-14T06:44:17Z",
          "NumViolations": 2,
          "ViolationComments": "RC : EFO PARAMETER TOO HIGH CA : ADJUST CURRENT PARAMETER FROM 60 TO 59 TECH : DITA",
          "ViolationList": "1,3",
          "PrimaryViolation": "1",
          "PrimaryViolationComments": "RC : EFO PARAMETER TOO HIGH CA : ADJUST CURRENT PARAMETER FROM 60 TO 59 TECH : DITA",
          "SampleMean": 23.7,
          "SampleMin": 21.0,
          "SampleMax": 25.4,
          "SampleMedian": 24.1,
          "SampleStdev": 1.62326830807479,
          "SampleQ1": 23.9,
          "SampleQ3": 24.1,
          "SampleSize": 5
        },
        {
          "Seqnr": 3,
          "SampleId": 240184639,
          "Value": 21.0,
          "is_flagged": "N",
          "SampleTimestamp": "2023-11-14T13:03:13Z",
          "SampleTimestampUtc": "2023-11-14T06:03:13Z",
          "CreatedTimestamp": "2023-11-14T13:03:14Z",
          "CreatedTimestampUtc": "2023-11-14T06:03:14Z",
          "UpdatedTimestamp": "2023-11-14T13:44:17Z",
          "UpdatedTimestampUtc": "2023-11-14T06:44:17Z",
          "NumViolations": 2,
          "ViolationComments": "RC : EFO PARAMETER TOO HIGH CA : ADJUST CURRENT PARAMETER FROM 60 TO 59 TECH : DITA",
          "ViolationList": "1,3",
          "PrimaryViolation": "1",
          "PrimaryViolationComments": "RC : EFO PARAMETER TOO HIGH CA : ADJUST CURRENT PARAMETER FROM 60 TO 59 TECH : DITA",
          "SampleMean": 23.7,
          "SampleMin": 21.0,
          "SampleMax": 25.4,
          "SampleMedian": 24.1,
          "SampleStdev": 1.62326830807479,
          "SampleQ1": 23.9,
          "SampleQ3": 24.1,
          "SampleSize": 5
        },
        {
          "Seqnr": 4,
          "SampleId": 240184639,
          "Value": 25.4,
          "is_flagged": "N",
          "SampleTimestamp": "2023-11-14T13:03:13Z",
          "SampleTimestampUtc": "2023-11-14T06:03:13Z",
          "CreatedTimestamp": "2023-11-14T13:03:14Z",
          "CreatedTimestampUtc": "2023-11-14T06:03:14Z",
          "UpdatedTimestamp": "2023-11-14T13:44:17Z",
          "UpdatedTimestampUtc": "2023-11-14T06:44:17Z",
          "NumViolations": 2,
          "ViolationComments": "RC : EFO PARAMETER TOO HIGH CA : ADJUST CURRENT PARAMETER FROM 60 TO 59 TECH : DITA",
          "ViolationList": "1,3",
          "PrimaryViolation": "1",
          "PrimaryViolationComments": "RC : EFO PARAMETER TOO HIGH CA : ADJUST CURRENT PARAMETER FROM 60 TO 59 TECH : DITA",
          "SampleMean": 23.7,
          "SampleMin": 21.0,
          "SampleMax": 25.4,
          "SampleMedian": 24.1,
          "SampleStdev": 1.62326830807479,
          "SampleQ1": 23.9,
          "SampleQ3": 24.1,
          "SampleSize": 5
        },
        {
          "Seqnr": 5,
          "SampleId": 240184639,
          "Value": 23.9,
          "is_flagged": "N",
          "SampleTimestamp": "2023-11-14T13:03:13Z",
          "SampleTimestampUtc": "2023-11-14T06:03:13Z",
          "CreatedTimestamp": "2023-11-14T13:03:14Z",
          "CreatedTimestampUtc": "2023-11-14T06:03:14Z",
          "UpdatedTimestamp": "2023-11-14T13:44:17Z",
          "UpdatedTimestampUtc": "2023-11-14T06:44:17Z",
          "NumViolations": 2,
          "ViolationComments": "RC : EFO PARAMETER TOO HIGH CA : ADJUST CURRENT PARAMETER FROM 60 TO 59 TECH : DITA",
          "ViolationList": "1,3",
          "PrimaryViolation": "1",
          "PrimaryViolationComments": "RC : EFO PARAMETER TOO HIGH CA : ADJUST CURRENT PARAMETER FROM 60 TO 59 TECH : DITA",
          "SampleMean": 23.7,
          "SampleMin": 21.0,
          "SampleMax": 25.4,
          "SampleMedian": 24.1,
          "SampleStdev": 1.62326830807479,
          "SampleQ1": 23.9,
          "SampleQ3": 24.1,
          "SampleSize": 5
        }
      ],
      "CFComment": "FOL",
      "FourDReport": "*",
      "Data_flat_Limits": {
        "CKC_ID": "519",
        "MeasurementSpecLimits": {
          "SpecHighEnabled": "Y",
          "SpecLowEnabled": "Y",
          "spec_high": 24.0,
          "spec_low": 9.0,
          "target": 16.5
        },
        "ControlLimits": {
          "ctrl_mean_low": 11.25,
          "CtrlMeanLowEnabled": "Y",
          "ctrl_mean_target": 16.5,
          "CtrlMeanTargetEnabled": "Y",
          "ctrl_mean_high": 21.75,
          "CtrlMeanHighEnabled": "Y",
          "CtrlLowEnabled": "N",
          "CtrlHighEnabled": "N",
          "CtrlStdevLowEnabled": "N",
          "CtrlSigmaCenterEnabled": "N",
          "CtrlRangeCenterEnabled": "Y",
          "CtrlCenterEnabled": "N",
          "CtrlStdevHighEnabled": "N",
          "CtrlRangeLowEnabled": "N",
          "ctrl_range_target": 0.0,
          "ctrl_range_high": 15.0,
          "CtrlRangeHighEnabled": "Y"
        }
      },
      "Data_flat_MeasurementAggregates": {
        "exec_count": 5,
        "base_count": 5,
        "flagged_count": 0,
        "mean": 23.7,
        "median": 24.1,
        "stdev": 1.4518953130305225,
        "range": 4.399999999999999,
        "min": 21.0,
        "max": 25.4,
        "q02": 21.232,
        "q05": 21.580000000000002,
        "q25": 23.9,
        "q75": 24.1,
        "q95": 25.139999999999997,
        "q98": 25.296,
        "NumViolations": 2,
        "ViolationComments": "RC : EFO PARAMETER TOO HIGH CA : ADJUST CURRENT PARAMETER FROM 60 TO 59 TECH : DITA",
        "ViolationList": "1, 3",
        "PrimaryViolation": "1",
        "PrimaryViolationComments": "RC : EFO PARAMETER TOO HIGH CA : ADJUST CURRENT PARAMETER FROM 60 TO 59 TECH : DITA",
        "Samples": "240184639"
      },
      "ParameterName": "GoldBallHeightBump",
      "ParameterUnit": "um",
      "ChannelId": "68003",
      "SourceDataLevel": "C",
      "channel_name": "Ball_Height AU 30um Bump",
      "RvStoreFlag": "Y",
      "created_timestamp_utc": "2023-11-14T06:03:14Z",
      "created_timestamp": "2023-11-14T13:03:14Z",
      "updated_timestamp_utc": "2023-11-14T06:44:17Z",
      "updated_timestamp": "2023-11-14T13:44:17Z"
    }
  ],
  "Document": {
    "Version": "1.0",
    "Type": "ProcessControl",
    "Repetition": {
      "Id": "0",
      "IdBaseValue": "2023-11-14T06:44:17Z"
    }
  },
  "ProductionAction": {
    "Id": "SPACEAGGED2:BAT:BE:BE1I:Wire_Bond1:",
    "Type": "SpaceAggregation"
  },
  "Item": {
    "Id": "Measlot:",
    "IdSystemName": "Measlot",
    "Type": "Lot"
  },
  "SystemLog": {
    "UniqueSourceKey": "BAT:BE:RU243389B26:BE1I:Wire_Bond1:GoldBallHeightBump:68003:240184639:519",
    "ExtractedTimestampUtc": "0001-01-01T00:00:00",
    "EnqueuedTimestampUtc": "0001-01-01T00:00:00",
    "DocCreatedTimestampUtc": "2023-11-14T13:03:13Z",
    "QueueOffset": 0,
    "QueuePartition": 0
  },
  "SearchPatterns": {
    "SpaceKey": "Measlot::SPACEAGGED2:BAT:BE:BE1I:Wire_Bond1::ProcessControl:1.0",
    "TimeGroup": "0",
    "SiteKey": "BAT"
  }
}